### ISSUE TITLE
[Jira] It does not allow searching for Epics

### ISSUE THREAD

**Issue by puneet-ekline** [raycast/extensions#12646]
### Extension

https://www.raycast.com/raycast/jira

### Raycast Version

1.75.1

### Description

When creating a jira issue using the extension, it does not allow searching for Epics. It always shows No epic

### Steps To Reproduce

1. Create a jira issue using the extension. 
2. Fill our the details
3. Search for a valid epic name

### Current Behaviour

It show no epic found

### Expected Behaviour

It should show the epic names

**Comment by thomaslombart** [comment]
Could you share a recording? It'll make it easier to see what's happening.

**Comment by puneet-ekline** [comment]
https://github.com/raycast/extensions/assets/135100789/a8fa42a2-fe53-4b48-af48-4fffa6727d22

Here is quick recording.

**Comment by thomaslombart** [comment]
Does it happen for all of your Jira projects?

**Comment by luarmr** [comment]
> Does it happen for all of your Jira projects?
^^^ This

It shows only the epics created in the current project where I am creating the issue, as the project ID is used to retrieve the list of epics. Even when possible, I used it for cross-team projects, which is not usual.

https://github.com/raycast/extensions/blob/63ed284e297dbbf9012e384d7829d96b47c9eb24/extensions/jira/src/components/FormParentDropdown.tsx#L20

**Comment by dipakparmar** [comment]
same, not seeing any epics, previously I was able to.

**Comment by ishassan** [comment]
The issue is happening consistently to me. I don't see Epics at all in search, whether within the correct intended project or not.

It used to work before, now not anymore.

**Comment by ishassan** [comment]
Oh, I think I know what's going on. The Epic autocomplete uses the api `/rest/api/3/issue/picker`. The API specs, according to the [docs](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-search/#api-group-issue-search), are defined as:
> Returns lists of issues matching a query string. Use this resource to provide auto-completion suggestions when the user is looking for an issue using a word or string.

> This operation returns two lists:

>    History Search which includes issues from the user's history of created, edited, or viewed issues that contain the string in the query parameter.
>    Current Search which includes issues that match the JQL expression in currentJQL and contain the string in the query parameter.

The extension seems to search for Epics within the "History Search" not the "Current Search", so it's only returning Epics that I interacted with. This matches with what I reported above that I'm only seeing Epics I created. It's actually Epics that I viewed or interacted with as well.

Interestingly, some of the Epics I'm missing are ones I already interacted with before, which makes me think Jira is keeping some retention limit on the user's history, and eventually old history is deleted and not retrievable via History Search anymore.

A simple workaround would be to just go and open all the Epics I'm interested in. However, I'm happy to make a PR to make us switch from History Search to Current Search if this is what we want. It makes sense to me especially assuming the Jira retention thing is correct. I tried it locally and seems to work.

Let me know what you think here and how you want to proceed.
